dev/reference/garbage-collection/configuration.md

```diff
@@ -0,0 +1,65 @@
+---
+title: GC 配置
+category: reference
+---
+
+# GC 配置
+
+TiDB 的 GC 相关的配置存储于 `mysql.tidb` 系统表中，以下列出这些参数及其含义。可以通过 SQL 语句对这些参数进行查询和更改。例如，如果需要将 GC 调整为每 30 分钟执行一次，只需执行下列语句即可：
+
+```sql
+update mysql.tidb set VARIABLE_VALUE="30m" where VARIABLE_NAME="tikv_gc_run_interval";
+```
+
+> **注意：**
+>
+> `mysql.tidb` 系统表中除了下文将要列出列出的 GC 的配置以外，还包含一些 TiDB 用于储存部分集群状态（包括 GC 状态）的记录。请勿手动更改这些记录。其中，与 GC 有关的记录如下：
+>
+> - `tikv_gc_leader_uuid`，`tikv_gc_leader_desc` 和 `tikv_gc_leader_lease` 用于记录 GC leader 的状态和进行 GC leader 的选举
+> - `tikv_gc_last_run_time`：上次 GC 运行时间
+> - `tikv_gc_safe_point`：当前的 GC safe point
+
+## `tikv_gc_enable`
+
+- 控制是否启用 GC。
+- 默认值：`true`
+
+## `tikv_gc_run_interval`
+
+- 指定 GC 运行时间间隔。Duration 类型，使用 Go 的 Duration 字符串格式，如 `"1h30m"`，`"15m"` 等。
+- 默认值：10 分钟（`"10m0s"`）
+
+## `tikv_gc_life_time`
+
+- 每次 GC 时，保留数据的时限。Duration 类型。每次 GC 时将以当前时间减去该配置的值作为 safe point。
+- 默认值：10 分钟（`"10m0s"`）
+
+> **注意：**
+>
+> - `tikv_gc_life_time` 的值必须大于 TiDB 的配置中的 [`max-txn-time-use`](/reference/configuration/tidb-server/configuration-file/#max-txn-time-use) 的值至少 10 秒。
```

Comment:
I found that this configuration's document doesn't seems to exist. It's it ok to merge this? Or  should we change this document?